### PR TITLE
fix: add a "set" trap to the "screen" module proxy

### PR DESCRIPTION
**Pull request by alexeykuzmin**
#### Description of Change

Let's make it possible to stub `screen` methods for testing.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes:  Fixed "screen" methods to be reassignable again.

### CODE REVIEW

**Review by miniak — APPROVED** [review]

File: spec-main/api-screen-spec.ts [hunk 1]
```diff
@@ -1,7 +1,19 @@
...
+  describe('methods reassignment', () => {
+    after(() => {
+      sinon.restore();
+    });
+
+    it('works for a selected method', () => {
+      sinon.stub((screen as any), 'getPrimaryDisplay').returns(null);
+      expect(screen.getPrimaryDisplay()).to.be.null();
+    });
+  });
```
Comment (nornagon) [inline spec-main/api-screen-spec.ts:15]: i'd prefer we test this more directly, without involving a 3rd-party dependency. perhaps:
```suggestion

describe('screen module', () => {
  describe('methods reassignment', () => {
    it('works for a selected method', () => {
      const originalFn = screen.getPrimaryDisplay
      try {
        (screen as any).getPrimaryDisplay = () => null;
        expect(screen.getPrimaryDisplay()).to.be.null();
      } finally {
        (screen as any).getPrimaryDisplay = originalFn
      }
    });
  });
```

Comment (alexeykuzmin) [reply spec-main/api-screen-spec.ts]: @nornagon, why not, done.

**Review by codebytere — APPROVED** [review]

**Comment by 0501814314** [comment]
Mosa.homdi2013@gmail.com

**Comment by 0501814314** [comment]
Mosa.homdi2013@gmail.com

**Pull request merged**